### PR TITLE
Feat/edit projects readmes

### PR DESCRIPTION
**Pull request by Ldoppea**
Edit `REAMDE.md`, `CONTRIBUTIN.md` and `SECURITY.md` to add Cozy specific information.

~~Picture in README.md will come from this PR https://github.com/cozy/cozy-pass-web/pull/27 so it is broken in the preview~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,119 @@
...
+git clone https://github.com/cozy/cozy-ui.git
+cd cozy-ui
+yarn install
+yarn link
+```
+
+then go back to your app project and replace the distributed cozy-ui module with the linked one:
+
+```sh
+cd cozy-drive
```
Comment (Crash--) [inline README.md:75]:
```suggestion
cd cozy-pass-web
```

File: SECURITY.md [hunk 2]
```diff
@@ -1,45 +1,139 @@
...
+of security weaknesses in Cozy. We will respond in less than 72 hours.
+
+When a security flaw is found, the process is the following:
+
+-   Make a pull-request to fix (on our private git instance) and test it.
+-   Deploy the fix on cozycloud.cc
+-   Publish a new version, announce it on
+    [the forum](https://forum.cozy.io/c/latest-information-about-cozy-security)
+    as a security update and on the mailing-lists.
+-   15 days later, add the details on the forum.
```
Comment (Crash--) [inline SECURITY.md:139]: I don't think we should keep this file. It seems to come from the stack documentation right? If yes,  we should just have a link to it. Not copying pasting the content. It's hard to maintain this way.

File: README.md (hunk 1 above)
Comment (Ldoppea) [reply README.md]: Fixed

File: SECURITY.md (hunk 2 above)
Comment (Ldoppea) [reply SECURITY.md]: New proposal :
- https://github.com/cozy/cozy-pass-web/blob/95bab48a55783d5df1a7b6d851511d59742dde15/SECURITY.md
- https://github.com/cozy/cozy-pass-web/blob/95bab48a55783d5df1a7b6d851511d59742dde15/CONTRIBUTING.md

**Comment by Ldoppea** [comment]
PR https://github.com/cozy/cozy-pass-web/pull/27 has been merged, so screenshot is now displayed correctly.

Readable version can be found here : https://github.com/cozy/cozy-pass-web/blob/95bab48a55783d5df1a7b6d851511d59742dde15/README.md

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,41 +1 @@
...
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/bitwarden-desktop/localized.svg)](https://crowdin.com/project/bitwarden-desktop)
-
-We use a translation tool called [Crowdin](https://crowdin.com) to help manage our localization efforts across many different languages.
-
-If you are interested in helping translate the Bitwarden desktop app into another language (or make a translation correction), please register an account at Crowdin and join our project here: https://crowdin.com/project/bitwarden-desktop
-
-If the language that you are interested in translating is not already listed, create a new account on Crowdin, join the project, and contact the project owner (https://crowdin.com/profile/kspearrin).
-
-You can read Crowdin's getting started guide for translators here: https://support.crowdin.com/crowdin-intro/
+Please see https://github.com/cozy/cozy-stack/blob/master/docs/CONTRIBUTING.md
```
Comment (Crash--) [inline CONTRIBUTING.md:1]: I think we should create a `contributing` guide in the "front documentation". I think that stack is a bit different

**Review by Crash-- — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (Ldoppea) [reply CONTRIBUTING.md:1]: Where should we store this documentation? Maybe in cozy-ui? Or in a dedicated "cozy" repository?

Comment (Crash--) [reply CONTRIBUTING.md:1]: https://github.com/cozy/cozy.github.io/tree/dev/src/howTos/dev

Maybe we should create a `contritubing-to-frond-end.md` there? But we can remove this `contributing` file in the meantime?

**Review by Crash-- — APPROVED** [review]

**Comment by nono** [comment]
Ping @Ldoppea

**Pull request merged**